global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.1.302"
```

Comment:
2.1.506 is the latest 2.1 LTS version

Reply:
That's correct, but it is not installed at AppVeyor at the moment. So I've decided to use old good version :)

Reply:
See https://ci.appveyor.com/project/DenisIvanov/orleans-persistence-cassandra/build/job/ndl81gfruia9dor8